### PR TITLE
Fixed gocyclo for cli package

### PR DESCRIPTION
**Pull request by apourchet**

### CODE REVIEW

File: cli/build.go [hunk 1]
```diff
@@ -147,15 +119,54 @@ func (cmd *BuildFlags) postInit() error {
...
+
 	// Verify storage dir is not child of internal dir.
 	if pathutils.IsDescendantOfAny(cmd.StorageDir, []string{pathutils.DefaultInternalDir}) {
 		return fmt.Errorf("storage dir cannot be under internal dir %s",
 			pathutils.DefaultInternalDir)
 	}
+	return nil
+}
 
+func (cmd BuildFlags) initRegistryGlobals() error {
```
Comment (evelynl94) [inline cli/build.go]: initRegistryConfig

File: cli/build.go [hunk 2]
```diff
@@ -147,15 +119,54 @@ func (cmd *BuildFlags) postInit() error {
...
+	if cmd.RegistryConfig == "" {
+		// TODO(pourchet): Shouldn't we do this regardless of if a registry config was passed?
```
Comment (evelynl94) [inline cli/build.go]: well map doesnt have an order and .* is dangerous...

Comment (apourchet) [reply cli/build.go]: That's fair

File: cli/utils.go [hunk 3]
```diff
@@ -108,18 +105,19 @@ func (cmd BuildFlags) saveImage(imageName image.Name, imageStore storage.ImageSt
 }
 
 // cleanManifest removes specified image manifest from local filesystem.
-func (cmd BuildFlags) cleanManifest(imageName image.Name, imageStore storage.ImageStore) error {
+func (cmd BuildFlags) cleanManifest(imageName image.Name) error {
 	repo, tag := imageName.GetRepository(), imageName.GetTag()
-	if err := imageStore.Manifests.DeleteStoreFile(repo, tag); err != nil && !os.IsNotExist(err) {
```
Comment (evelynl94) [inline cli/utils.go]: This looks alright... Is goreport complaining about this line?

Comment (apourchet) [reply cli/utils.go]: The line became > 100 characters when i added the `cmd.` in front of the imageStore, i'd rather split the if into a new line now if that's not an issue with you.

**Pull request merged**